redisinsight/api/provider-interfaces/models/ca-certificate.ts

```diff
@@ -0,0 +1,10 @@
+// todo: discuss
+// probably not needed for cloud
+// if needed - we must introduce userId here
+export class CaCertificate {
```

Comment:
The `CaCertificate` applies to all Redis databases and is provided by Redis itself. 
We will need the model, but in the provider we only need the `get` which will always return the same `CaCertificate` regardless of parameter.